server/routes/species.js

```diff
@@ -1,17 +1,41 @@
-import db from "../db/DbConnection";
+import db from "../db/DbConnection.js";
 import express from "express";
 const router = express.Router();
 /* GET users listing. */
 
 router.get('/', async function (req, res) {
 
   try {
-    const species= await db.any('SELECT * FROM species ORDER BY id', [true]);
+    const species = await db.any('SELECT * FROM species ORDER BY id', [true]);
     res.send(species);
   } catch (e) {
     console.log(e);
     return res.status(400).json({ e });
   }
 });
 
+//post request
+router.post('/', async (req, res) => {
+  const species = {
+    id: req.body.id,
+    common_name: req.body.common_name,
+    scientific_name: req.body.scientific_name,
+    population: req.body.population,
+    conservation_status: req.body.conservation_status,
+    created_on: req.body.created_on,
+  }
+  console.log(species);
+  try {
+    const createdSpecies = await db.one(
```

Comment:
Like do you think you're having a problem with the query? Or making the post request?

At a birds eye view this all looks good so please share errors or comment on specific parts of code with questions.

Reply:
I thought I had an issue getting the input to the database but I got that working. So far all changes are good to go, would like if we can go over a join table